nix/lib/node_modules.nix

```diff
@@ -0,0 +1,28 @@
+{ npmlock2nix
```

Comment:
What's this file for? I think it's just convenience wrappers around `npmlock2nix`, right?

Reply:
This file is exposed as `plutus.buildNodeModules`. The idea is for this to encapsulate building a `node_modules` folder that can be symlinked into a Purescript build.

Maybe it's fair to call it a npmlock2nix wrapper since that's what we are using now, but i mostly just want the npm/node_modules business out of the purescript build. Should we ever (hopefully not) need to switch to something else *again* we would then ideally only need to change node_modules.nix since in the purescript build the requirement of just symlinking in the `node_modules` folder should stay the same.